RELEASE.md

```diff
@@ -34,6 +34,36 @@ Current Release Team:
 - Suzen Fylke - @codesue
 
 ## Past Releases
+
```

Comment:
Nice! Let's move these notes to https://github.com/tensorflow/model-card-toolkit/releases after merging changes to `version.py`. The past releases section is mainly here to accommodate versions  older than 1.3.2 since some don't have a corresponding GitHub release.

Reply:
Thank you @codesue. I removed the release notes - happy to create them on Github when we release the production package